README.md

```diff
@@ -0,0 +1,34 @@
+# website_support_pdf
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg) ](LICENSE.md)  
+
+This is an addon module for **odoo 12**, that
+Enables the *website_support* module to export the ticket info, aswell as the Chatter section in a **PDF** format.  
+* [Why?](#use-case)
+* [Requirements](#requirements)  
+* [How to Install](#installing)  
+* [TODOS](#todos)
+### Use Case
+ I was in the need of printing the info in the ticket with all the logs aswell as the chatter section.
+ 
+### Requirements
+* [Website Support module](https://apps.odoo.com/apps/modules/12.0/website_support/)
+* [Odoo 12](https://www.odoo.com/odoo-12-release-notes)
+
+### Installing  
+
+Make sure you clone the repo into the addons folder and then install like any other odoo module
+```
+git clone https://github.com/yokodev/website_support_pdf.git
+```
+You can also install it from the command line like so:
+
+```
+./odoo/odoo-bin -c myOdoo.conf -i website_support_pdf
+```
+
+Install like any other odoo module in Odoo
+
+### Todos
+* PR are Welcome Contribuiting Guide
```

Comment:
1. **PRs** intead of just **PR**
2. ...are Welcome**,** Contribuiting Guide is also required if needed...

Reply:
Closing down because needs some typos